balancer-js/src/modules/sor/token-price/coingeckoTokenPriceService.ts

```diff
@@ -31,7 +32,10 @@ export class CoingeckoTokenPriceService implements TokenPriceService {
       },
     });
 
-    if (data[tokenAddress.toLowerCase()][this.nativeAssetId] === undefined) {
+    if (
+      isUndefined(data[tokenAddress.toLowerCase()]) ||
+      isUndefined(data[tokenAddress.toLowerCase()][this.nativeAssetId])
```

Comment:
Do we really need loadash for this? Why not simply check for undefined?

```suggestion
      data[tokenAddress.toLowerCase()] === undefined ||
      data[tokenAddress.toLowerCase()][this.nativeAssetId] === undefined
```